app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -21,7 +22,8 @@ const Notes = () => {
   const [notes, setNotes] = useState([]);
 
   useEffect(() => {
-    fetchNotes();
+    setNotes(NOTES_TABLE_ROW_DATA);
+    setLoading(false);
```

Comment:
- Can you justify the need to set the notes with a constant data within the `useEffect`?
- Also what's the need for setting `loading` as false within this useEffect?

Reply:
@yedhink I thought of retaining the same logic used in the code.
I guess a better approach would be to assign it directly inside useState when the states are declared and set the initial value of loading to false.

Reply:
>I thought of retaining the same logic used in the code.

If the existing logic doesn't fit in with the requirements of the challenge, then by all means rectify rather than retaining.